services/app/src/components/loginPage/LoginForm.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext, useState } from 'react';
+import { useNavigate } from 'react-router';
+import { WsContext } from '../../contexts/ws-context';
+import './loginPage.css'
+import userInputIcon from './SVGs/userInputIcon.svg';
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+  const navigateUrl = '/game/test';
```

Comment:
This value is used in two places already. Move it to some constants file and reuse it. Same applies to all the paths in the application. Go through them as well.